tests/processing/test_create_plate_validator.py

```diff
@@ -540,16 +626,28 @@ def test_validate_adds_error_when_plate_coordinate_column_invalid(subject, creat
     add_error.assert_has_calls(
         [
             call(
-                RABBITMQ_CREATE_FEEDBACK_ORIGIN_SAMPLE,
-                "Field value does not match regex (^[A-H](?:0?[1-9]|1[0-2])$).",
-                samples[0][FIELD_SAMPLE_UUID].decode(),
-                FIELD_PLATE_COORDINATE,
+                CreatePlateError(
+                    origin=RABBITMQ_CREATE_FEEDBACK_ORIGIN_SAMPLE,
+                    description="Field value does not match regex (^[A-H](?:0?[1-9]|1[0-2])$).",
+                    long_description=(
+                        "Field 'plateCoordinate' on sample '0aae6004-8e01-4f7a-9d50-91c51052813f' contains the value "
+                        f"'A{invalid_column}' which doesn't match the expected format for values in this field."
+                    ),
+                    sample_uuid=samples[0][FIELD_SAMPLE_UUID].decode(),
+                    field=FIELD_PLATE_COORDINATE,
+                )
             ),
             call(
-                RABBITMQ_CREATE_FEEDBACK_ORIGIN_SAMPLE,
-                "Field value does not match regex (^[A-H](?:0?[1-9]|1[0-2])$).",
-                samples[1][FIELD_SAMPLE_UUID].decode(),
-                FIELD_PLATE_COORDINATE,
+                CreatePlateError(
+                    origin=RABBITMQ_CREATE_FEEDBACK_ORIGIN_SAMPLE,
+                    description="Field value does not match regex (^[A-H](?:0?[1-9]|1[0-2])$).",
+                    long_description=(
+                        "Field 'plateCoordinate' on sample 'a9071f9c-0e3c-42c9-bef2-1045e827f9df' contains the value "
```

Comment:
I see the advantage of keeping strict the comparison with the uuids, but maybe for legibility we can put all uuids together somewhere and refer them? Or could we use a different string for human readable than the uuid? (if is not validated) or a fake uuid 0000000001, 000000002, etc

Reply:
I was feeling this at the time but got so deep into it that I decided to keep going.  I agree that we should use some simpler UUIDs at the expense of them not being realistic data for the test.